roles/openstack_helm_neutron/meta/main.yml

```
@@ -25,5 +25,11 @@ galaxy_info:
 
 dependencies:
   - role: defaults
-  - role: atmosphere
+  - role: openstack_helm_endpoints
+    vars:
+      openstack_helm_endpoints_chart: neutron
+  - role: upload_helm_chart
+    vars:
+      upload_helm_chart_src: "{{ openstack_helm_neutron_helm_chart_path }}"
+      upload_helm_chart_dest: "{{ openstack_helm_neutron_helm_chart_ref }}"
   - role: openstacksdk
```

same here. `openstacksdk` should be ahead of `openstack_helm_endponits` role
```suggestion
  - role: openstacksdk
  - role: openstack_helm_endpoints
    vars:
      openstack_helm_endpoints_chart: neutron
  - role: upload_helm_chart
    vars:
      upload_helm_chart_src: "{{ openstack_helm_neutron_helm_chart_path }}"
      upload_helm_chart_dest: "{{ openstack_helm_neutron_helm_chart_ref }}"
```